src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -59,16 +64,27 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $container = $this->getApplication()->getKernel()->getContainer();
         $serviceId = $input->getArgument('name');
+        $format = $input->getOption('dump_format');
+        $transitionType = $input->getOption('puml_transition_format');
```

Comment:
This var declaration could be (re)moved to line 81

Reply:
moved